### PR TITLE
Coverage (starting small)

### PR DESCRIPTION
**Pull request by sdhunt**
Hi Robin, 

I'd like to start contributing to the RealmSpeak codebase.

Thought I'd start out small, by cleaning up ForceVector, and creating unit tests for it.

Will continue with classes in the `general.graphics` package if you approve.

Cheers,
~Simon

### CODE REVIEW

File: general/utility/general_graphics/source/com/robin/general/graphics/ForceVector.java [hunk 1]
```diff
@@ -3,105 +3,210 @@
...
+    public static void main(String[] args) {
+        System.out.println(new ForceVector(1.0, 0.0));
+        System.out.println(new ForceVector(1.0, 90.0));
+        System.out.println(new ForceVector(1.0, 180.0));
+        System.out.println(new ForceVector(1.0, 270.0));
+
+        ForceVector fv1 = new ForceVector(1.0, 0.0);
+        ForceVector fv2 = new ForceVector(1.0, 90.0);
+        System.out.println(fv1.add(fv2));
+    }
```
Comment (dewkid) [inline general/utility/general_graphics/source/com/robin/general/graphics/ForceVector.java:211]: Did the spacing change here?  Tabs to spaces?

BTW - ForceVector isn't used anywhere in RealmSpeak.  I think this is a utility class I had in here for something else, and it just came along for the ride.  Still, I appreciate you making an effort to add some code coverage!

**Comment by dewkid** [comment]
I know its like TWO YEARS after the fact, and I imagine you've long since lost hope of me ever looking at this, but I'm going to try and be better about checking out this page (new years resolution?).  Also, as I use GIT now regularly in my job, I'm MUCH more familiar with the interface and how git works to actually be of some help here.

**Comment by dewkid** [comment]
I'm not a fan of switching from tabs to spaces, if that's what happened with ForceVector class, but otherwise, the changes look good to me. :-)

**Comment by sdhunt** [comment]
Hey Robin, 

My life has become pretty hectic project-wise, and working on _Realmspeak_ is pretty deep in my stack of things to do; I just don't know when (if?) I'll get back to it.

You can abandon this PR if you wish.

We can agree to differ on tabs vs. spaces.

All the best.
~Simon

**Comment by dewkid** [comment]
Hi Simon,

Yeah, I feel kind of bad about it.  I didn’t even know what a “PR” was when I first set this up, so didn’t know what to do with it.  Now I deal with them every day on the job, so I thought I should at least come back and look at it.

BTW – we use spaces at work too, but I’m more a tab-guy. ;-)

In any case, I wasn’t expecting you to respond, and didn't expect you were still working on it, so thanks for the response.

Robin

ps - another reason I was looking at it, is because someone reported a bug with Melt Into Mist spell, and it made me think it might be worth fixing after all these years... I don't know, we'll see!

**Pull request closed without merging**